x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Add documents to adjust settings');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/result-settings-guide.html')
+      expect.stringContaining(`${docLinks.appSearchResultSettings}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchResultSettings)
```
No need for a template literal here